v22.1/experimental-features.md

```diff
@@ -55,6 +55,10 @@ To relocate just the lease without moving the replicas, run a statement like the
 > ALTER TABLE t EXPERIMENTAL_RELOCATE LEASE SELECT 1, 'foo';
 ~~~
 
+{{site.data.alerts.callout_info}}
+<span class="version-tag">New in v22.1:</span> If you prefer to use an approach to relocating replicas and leases based on store IDs, see the [`ALTER RANGE ... RELOCATE`](alter-range-relocate.html) statement.
```

Comment:
replace store IDs with range IDs the benefit is really you get to use range ids instead of keys.

Reply:
ok - updated, thanks!